_posts/hi/newsletters/2022-08-17-newsletter.md

```diff
@@ -0,0 +1,117 @@
+  इसमें API परिवर्तन को तोड़ना शामिल है, लेकिन कई नई सुविधाएँ और बग फिक्स भी शामिल हैं, जिसमें
+  [कॉम्पैक्ट ब्लॉक रिले][topic compact block relay] डेटा संरचनाओं ([BIP152][])
+  और [Taproot][topic Taproot] ​​और [पीएसबीटी][topic psbt] समर्थन सुधार शामिल है।
+
+- [Core Lightning 0.12.0rc2][] इस लोकप्रिय LN नोड कार्यान्वयन के अगले प्रमुख संस्करण के लिए एक रिलीज उम्मीदवार है।
+
+## उल्लेखनीय कोड और दस्तावेज़ीकरण परिवर्तन
+
+*इस सप्ताह [Bitcoin Core][bitcoin core repo], [Core Lightning][core lightning repo], [Eclair][eclair repo],
+[LDK][ldk repo], [LND][LND Repo] में उल्लेखनीय परिवर्तन। [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Rust Bitcoin][Rust bitcoin repo],
+[BTCPay Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement Proposals (BIP)][BIPs repo],
+और [Lightning BOLTs][bolts repo]।*
+
+- [Bitcoin Core #23480][] [output script descriptor][topic descriptors] भाषा को एक `rawtr()`
+  डिस्क्रिप्टर के साथ अपडेट करता है ताकि उन मामलों में एक Taproot आउटपुट में उजागर कुंजी को
+  संदर्भित किया जा सके जहां या तो कुंजी का उपयोग बिना ट्वीक के किया जाता है (अनुशंसित नहीं है, [BIP341][bip341 internal] देखें) या जब आंतरिक कुंजी और स्क्रिप्ट ज्ञात न हों (जो असुरक्षित हो सकती हैं; विवरण के लिए PR टिप्पणियां या इस PR द्वारा जोड़े
+  गए दस्तावेज देखें)। हालांकि मौजूदा `raw()` डिस्क्रिप्टर का उपयोग करके उन मामलों में कुंजी को संदर्भित करना
+  पहले से ही संभव है, जो मुख्य रूप से UTXO के अपने डेटाबेस को स्कैन करने के लिए Bitcoin Core के `scantxoutset` RPC जैसे
+  उपकरणों के साथ उपयोग करने के लिए है, नया `rawtr()` डिस्क्रिप्टर अन्य मौजूदा डिस्क्रिप्टर फ़ील्ड का उपयोग करना
+  आसान बनाता है ताकि अतिरिक्त जानकारी को मुख्य मूल जानकारी जैसे Taproot आउटपुट के साथ जोड़ा जा सके। मुख्य
+  मूल जानकारी यह संकेत दे सकती है कि एक वैकल्पिक कुंजी पीढ़ी योजना का उपयोग किया जा रहा है, जैसे कि
+  [वैनिटी एड्रेस] बनाने के लिए वृद्धिशील ट्विकिंग [] या [गोपनीयता के लिए सहकारी ट्विकिंग][reusable taproot addresses]।
+
+- [Bitcoin Core #22751][] एक `simulaterawtransaction` RPC जोड़ता है जो अपुष्ट लेनदेन की एक सरणी को स्वीकार करता है और यह लौटाता है
+  कि वे लेनदेन वॉलेट की शेष राशि से कितना BTC जोड़ेंगे या घटाएंगे।
+
+- [Eclair #2273][] [प्रस्तावित][bolts #851] इंटरएक्टिव फंडिंग प्रोटोकॉल को लागू करता है जहां दो LN नोड्स एक नया भुगतान चैनल खोलने के लिए अधिक निकटता से समन्वय करते हैं।। इंटरएक्टिव फंडिंग को लागू करने से Eclair [डुअल फंडिंग][topic dual funding] के समर्थन
+  के करीब पहुंच जाता है, जो उस चैनल में भाग लेने वाले नोड्स में से किसी एक द्वारा नए चैनल में फंड का
+  योगदान करने की अनुमति देता है। इस सप्ताह [Eclair #2247][] में दोहरी फंडिंग के लिए अतिरिक्त तैयारी को भी मिला दिया गया।
+
+- [Eclair #2361][] को [BOLTs #996][] द्वारा प्रस्तावित `htlc_maximum_msat` फ़ील्ड को शामिल करने के लिए चैनल अपडेट की आवश्यकता शुरू होती है (देखें [न्यूज़लेटर #211][news211 bolts996])।
+
+- [LND #6810][] लगभग सभी वॉलेट की स्वचालित रूप से जेनरेट की गई आउटपुट स्क्रिप्ट में [Taproot][topic taproot] आउटपुट
```

Comment:
There was a ~space character between the brackets here causing a rending (but not build) issue. I pushed a small change to remove it.